### ISSUE TITLE
Prostration

### ISSUE THREAD

**Issue by cmrn-rhi** [obophenotype/human-phenotype-ontology#7119]
**Preferred term label:**
prostration

**Synonyms**
extreme exhaustion

**Definition (free text, please give PubMed ID)**
Complete physical or mental exhaustion. [source: https://www.merriam-webster.com/dictionary/prostration]

_- Appears frequently in PubMed publications, but not clearly defined._

**Parent term (use hpo.jax.org/app)**
HP:0012378 - Fatigue

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
COVID-19

**Your nano-attribution (ORCID)**
0000-0002-9578-0788

**Database cross reference**
[SNOWMEDCT_US:84946008](https://www.findacode.com/snomed/84946008--extreme-exhaustion.html?hl=prostration)

**Comment by pnrobinson** [comment]
I am not sure that a separate term for prostration is a good idea, because it is hard to clearly separate it from Fatigue/Asthenia, and the English word also has lots of other meanings (e.g. The act or condition of prostrating (lying flat) oneself, as a sign of humility.). 

I think it is more closely related to asthenia https://hpo.jax.org/app/browse/term/HP:0025406

Can I ask if in the articles you are reading the authors distinguish between Asthenia (which is often called 'weakness' in publications), fatigue, and prostration? For instance, are these three concepts separate rows in a table of findings?

**Comment by pnrobinson** [comment]
I will add this as a related synonym to asthenia.

**Comment by cmrn-rhi** [comment]
> Can I ask if in the articles you are reading the authors distinguish between Asthenia (which is often called 'weakness' in publications), fatigue, and prostration? For instance, are these three concepts separate rows in a table of findings?

Unfortunately yes, and example of which would be: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7097833/
However, in this case it is differentiating from "severe asthenia" so perhaps it would be synonymous with regular "asthenia".

I'm going to move forward with your recommendation of asthenia and return only if this proves to be a problem for our users.
